### PR TITLE
Documents search, limit, offset query parameters when listing resources

### LINKED ISSUE #165: Include search and pagination options in the documented parameters for "View resources" route

### PR DESCRIPTION
**Pull request by ryanprior**
Closes #165 

#### What does this pull request do?
Explains how to narrow your resource searches using three query parameters.

#### What background context can you provide?
In order to query Conjur more efficiently, API users should use query parameters instead of asking for the whole list and filtering client-side. This PR documents those useful query parameters.

#### Where should the reviewer start?
`apidocs/src/list_resources.md`

#### How should this be manually tested?
Build the apidocs as per README.md and view them in your browser. Check the "List resources" subheading.

#### Link to build in Jenkins (if appropriate)
https://jenkins.conjur.net/job/cyberark--conjur/job/doc%252Fresource-show-limit-offset-search%252F165/

### CODE REVIEW

File: apidocs/src/list_resources.md [hunk 1]
```diff
@@ -1,16 +1,37 @@
...
 Lists resources within an organization account.
 
+If a `kind` query parameter is given, the results will be narrowed to only
+resources of that kind. Likewise, if `search` is given, it will be narrowed to
+only resources with names or annotations containing the search term.
+
+If a `limit` is given, no more than that number of results will be returned.
+Providing an `offset` will skip a number of resources before returning the rest.
+These two parameters can be combined to page through results.
+
```
Comment (ryanprior) [inline apidocs/src/list_resources.md]: Done

**Comment by ryanprior** [comment]
@mizziness I documented the "count" feature as per your suggestion.

File: apidocs/src/list_resources.md [hunk 2]
```diff
@@ -1,16 +1,37 @@
 ## List resources [/resources]
 
-### List resources [GET /resources/{account}{?kind}]
+### List resources [GET /resources/{account}{?kind}{?search}{?limit}{?offset}]
 
 Lists resources within an organization account.
 
+If a `kind` query parameter is given, the results will be narrowed to only
+resources of that kind. Likewise, if `search` is given, it will be narrowed to
+only resources with names or annotations containing the search term.
```
Comment (ryanprior) [inline apidocs/src/list_resources.md]: Okay, this is more thoroughly documented now.

**Comment by kgilpin** [comment]
Can you make it clear that it's the annotation vaules that are being searched? Then I think we are good to go.

**Comment by ryanprior** [comment]
@kgilpin I clarified that text.

**Pull request merged**